src/components/Success.vue

```diff
@@ -1,23 +1,30 @@
 <template>
-    <div class="success center">
-        <div class="icon-checkmark-circle"></div>
-        <h1 v-html="parsedText"></h1>
+    <div class="success nq-bg-green">
+        <div class="nq-icon checkmark-circle"></div>
+        <h1 v-html="parsedText" class="nq-h1"></h1>
         <div style="flex-grow: 1;"></div>
-        <button @click="$emit('continue')">Back to {{ appName }}</button>
+
+        <button v-if="buttonText" @click="$emit('continue')" class="nq-button green inverse" :disabled="disabled">{{ buttonText }}</button>
+        <button v-else @click="$emit('continue')" class="nq-button green inverse" :disabled="disabled">Back to {{ appName }}</button>
```

Comment:
Could be done as getter instead of v-if.

Reply:
If you mean that the button text could be a getter, then yes, I also though that, until I noticed that then we would have to escape `appName` manually.

Reply:
would
```
get getButtonText() {
  return (buttonText) ? buttonText : 'Back to " + appName;
}
```
not get escaped when rendered as `{{ getButtonText }}`

Reply:
You are of course right xD